shared/chat/conversation/input-area/normal/index.js

```diff
@@ -59,49 +50,56 @@ class Input extends React.Component<InputProps> {
   }
 
   componentDidMount = () => {
+    // Set lastQuote so we only inject quoted text after we mount.
+    this._lastQuote = this.props._quoteCounter
```

Comment:
:sheep: Should this prop not have an `_` since it's used in the component? (same for `_editText` and others)

Reply:
Yeah, probably. Fixed!